torch/csrc/jit/passes/vulkan_rewrite.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <torch/csrc/jit/api/module.h>
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch {
+namespace jit {
+TORCH_API void vulkanInsertPrePackedOps(std::shared_ptr<Graph>& graph);
+TORCH_API void vulkanInsertPrePackedOps(script::Module& module);
+TORCH_API void vulkanFusePrePackedConvWithClamp(script::Module& module);
+TORCH_API void vulkanFoldPrePackingOps(script::Module& module);
+TORCH_API c10::optional<script::Module> vulkanOptimizeForMobile(
+    const script::Module& module);
+} // namespace jit
+} // namespace torch
```

Comment:
Also can you add some tests for graph rewriting? You can follow this one: https://github.com/pytorch/pytorch/blob/master/test/test_xnnpack_integration.py#L380. This is just to make sure we are indeed doing the right transformation. Although the function correctness check might not be possible in CI we can at least check graph rewrite.